client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -86,6 +86,8 @@ export const CreateEncounterModal: FC = () => {
       status:
         date && DateUtils.isFuture(date)
           ? EncounterNodeStatus.Scheduled
+          : date && DateUtils.isPast(date)
+          ? EncounterNodeStatus.Missed
```

Comment:
cool! just a note: there is the plan to have a single Pending status and calculate Scheduled and Missed dynamically based on the current date.